src/Raven.Server/ServerWide/Commands/UpdateDatabaseCommand.cs

```diff
@@ -56,5 +56,10 @@ public static void EnsureTaskNameIsNotUsed(DatabaseRecord record, string name)
                 throw new RachisApplyException($"Task name `{name}` is already in use", e);
             }
         }
+
+        public virtual void AssertLicenseLimits(ServerStore serverStore, DatabaseRecord databaseRecord, ClusterOperationContext context)
```

Comment:
wonder if it better to make this an abstract, so it will force us to think whether we need to consider licensing or not

Reply:
I think abstract will be better. @grisha-kotler ?

Reply:
I agree

Reply:
Done